_schema.graphql

```diff
@@ -1617,6 +1617,8 @@ type ArtworkContextAuction implements Node {
     all: Boolean = false
   ): [SaleArtwork]
   sale_artworks_connection(
+    # List of sale artwork IDs to fetch
+    ids: [String]
```

Comment:
Small nit, shouldn't this be `ids: [ID]`?

Reply:
All existing `*_ids` arguments are `[String]`, but perhaps we could start using `[ID]`?

cc/ @alloy

Reply:
Yes please 👌

Reply:
Okay, updated the PR and now they are using `[ID]`.